tooling-server-configuration/src/main/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/arquillian/ArquillianConfigurationExtension.java

```diff
@@ -0,0 +1,19 @@
+package org.jboss.eap.qe.microprofile.tooling.server.configuration.arquillian;
+
+import org.jboss.arquillian.container.test.spi.RemoteLoadableExtension;
+import org.jboss.arquillian.core.spi.LoadableExtension;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.ConfigUtils;
+
+/**
+ * Extension which sets property descriptor in {@link ConfigUtils} class which is used in MP OpenAPI spec tests
```

Comment:
remove reference to MP OpenAPI....`which is used in MP OpenAPI spec tests`. I can see it in other parts as well so it will need clean up.

Reply:
Removed. I searched the code for "openapi" substring flavors and only found in `ManagementClientHelper`, which could or not need a refactoring now, it depends whether the desired solution is a common helper or a per spec helper. WDYT?

Reply:
I'm more for per spec helper as it aggregates configuration for given spec. Looking at `ManagementClientHelper` it could be renamed to `OpenApiManagementClientHelper`

Reply:
Done